### ISSUE TITLE
Local Adjustments Dehaze & Retinex tool: Basic vs Standard

### ISSUE THREAD

**Issue by chaimav** [Beep6581/RawTherapee#6540]
RawTherapee 5.8 3105, Windows 10
There is currently no difference between _Basic_ and _Standard_ modes for Local Adjustments Dehaze & Retinex tool.

Recommended solution is to either add some tools from _Advanced_ to _Standard_ or to eliminate _Standard_ entirely 

![dehaze standard](https://user-images.githubusercontent.com/47765782/182923620-c0713457-6a79-4c22-a417-4ca30e906b18.JPG)
![Dehaze basic](https://user-images.githubusercontent.com/47765782/182923624-de0c53e3-343b-47ff-bc59-632928bee680.JPG)

**Comment by Desmis** [comment]
@chaimav 
Yes I know, it is the only case, where Mask are only for Retinex, and Mask with Retinex are very different in conception with the others. Impossible to add to Standard.  And the 3 sliders are necessary.
But it is impossible (at least for me) to change only one selection "Basic", Standard", "Advanced"...

The only tool I can add, is "Retinex"...and very complex

Jacques

**Comment by chaimav** [comment]
@Desmis I understand your reasoning, but I would prefer to keep this issue open in the event  that the GUI can ever be reworked to have only 2 states.

**Comment by Desmis** [comment]
OK, but not by me...Too complex

Jacques

**Comment by Desmis** [comment]
@chaimav 
I tried to provide a (partial) solution to the problem of the complexity menu: basic - standard - advanced

Now the choice is "basic-basic-advanced" for "Dehaze - Retinex"... and of course no differences with the 2 "basic", and "advanced" allow Retinex
To go further is out of my competence

I have push a change in branch : la_retinex_complex

Jacques

**Comment by chaimav** [comment]
Thanks for the effort, but I am honestly not sure that this solution is really any better. Maybe we can wait for someone else to weigh in or even solicit input from pixls.us

**Comment by Desmis** [comment]
@chaimav 
I think, you will wait a long time. This part of the GUI is very specific
But no problem, if my poor solution is not good

Jacques
